opm/autodiff/BlackoilModelParameters.hpp

```diff
@@ -68,6 +81,13 @@ namespace Opm
         /// Try to detect oscillation or stagnation.
         bool use_update_stabilization_;
 
+        /// Whehter to use MultisegmentWell to handle multisegment wells
```

Comment:
Typo: "Whether" is correct. Also, include the rest of the commentary in the Doxygen doc by using triple slashes, but remove the next sentence first ("it is something temporary...").